tests/integration/__init__.py

```diff
@@ -6,7 +6,7 @@
 import pytest
 from betamax.cassette import Cassette
 
-from .utils import (
+from ..utils import (
```

Comment:
I don't think that this change is correct as there is a separate `utils.py` in `tests/integration/`

Reply:
Oh nevermind. I see that you relocated that file. Are we using that util for anything other than integration tests?

Reply:
We're not but the ruff checks gets mad about having a module not prefixed with `test_` in the test folders.

Reply:
Okay, that makes sense then.